src/main/resources/templates/signup2.html

```diff
@@ -5,36 +5,35 @@
 		<title>National Cybersecurity Laboratories - Login</title>
 		<object th:include="fragments/header :: header" th:remove="tag" />
 	</head>
-    <body class="mybg">
+	<body class="mybg registerback">
         <div th:replace="fragments/login_navigation :: loginnav"></div>
         <div class="container">
             <div class="row">
                 <div class="col-md-1"></div>
                 <div class="col-md-10">
-                    <div class="panel panel-default registration-panel">
-                        
-                        <div class="panel-heading">
-                            <h3 class="dashhead" th:text="#{signup.header}">Registration</h3>
-                        </div> <!-- panel-heading -->
-                        
+                    <div class="panel panel-default registration-panel2">
+						 <h3 class="dashhead loginpanel" th:text="#{signup.header}">Registration</h3>
+                      <!-- panel-heading -->
+						<div class="welcome">
+							<p> <b>Come join NCL. Let's set up your account.</b></p>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <b> tag by <strong>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)